src/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react'
+import NavbarRB from 'react-bootstrap/Navbar'
+import Nav from 'react-bootstrap/Nav'
+import Form from 'react-bootstrap/Form'
+import FormControl from 'react-bootstrap/FormControl'
+import Button from 'react-bootstrap/Button'
+import NavDropdown from 'react-bootstrap/NavDropdown'
+
+interface Props {
+  /** Determines the navbar background color */
+  bg?: 'light' | 'dark'
+  /** Determines the letters color. It should be combined with the background color (bg) */
+  variant?: 'light' | 'dark'
+}
+
+/**
+ * Used to redirect users to the main topics.
+ */
+class Navbar extends Component<Props, {}> {
+  render() {
+    return (
+      <NavbarRB
+        bg={this.props.bg ? this.props.bg : 'dark'}
+        variant={this.props.variant ? this.props.variant : 'dark'}
+      >
+        <NavbarRB.Brand href="#">{' HospitalRun'}</NavbarRB.Brand>
```

Comment:
We should be able to dynamically update the brand here. I'm thinking a prop called `brand`.

This would allow for hospital/clincs to insert their own branding.

Reply:
Also, we should probably support an image or text.